testcases/misc_tests/tok2tok_transport.c

```diff
@@ -664,6 +664,12 @@ CK_RV do_wrap_key_test(struct wrapped_mech_info *tsuite,
 
     case CKM_AES_KEY_GEN:
     case CKM_AES_XTS_KEY_GEN:
+        if ((is_ep11_token(slot_id1) || is_ep11_token(slot_id2))
+            && tsuite->wrapped_key_gen_mech.mechanism == CKM_AES_XTS_KEY_GEN){
+            testcase_skip("Skipping as AES XTS is supported only with protected keys");
+            rc = CKR_OK;
+        goto testcase_cleanup;
```

Comment:
Minor style issues (sorry that I did not notice this before):
- the `goto` should be indented.
- blank missing before `{` in line 668:  `){`
- I would usually put the `&&` at the end of the previous line.